core/tests/performance_framework/process_data.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Contains domain objects for storing page session stats as provided by a
+HTTP Archive file (also referred to as HAR).
+
+Selenium and Browsermob-proxy are used for capturing session information, such
+as load times and page size statistics. Timing statistics are retrieved directly
+from the browser console rather than the proxy server, as the latter is sluggish
+and gives inaccurate timings.
+"""
+
+import utils
+
+
+class PageSessionMetrics(object):
+    """Contains methods to process stats and provide performance metrics.
+
+    page_session_stats is a dictionary containing page load statistics from an
+    HTTP Archive.
+    (https://dvcs.w3.org/hg/webperf/raw-file/tip/specs/HAR/Overview.html)
+
+    page_session_timings is a dictionary containing metrics associated with page
+    loading and includes the following keys:
+
+        timing: maps to a dict containing the keys (all in milliseconds):
```

Comment:
Is is much clearer. Thanks! Nit: could you order the list of keys alphabetically? Otherwise it looks good to me.